### PR TITLE
Remove installation scripts

### PR DESCRIPTION
**Pull request by memeramita**
Removed installation scripts and updated the README instructions to running the commands directly from the command line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
-
-### 2. Install the plugin
-Click the link corresponding to your OS.
-#### [Windows](https://alleninstitute-my.sharepoint.com/:u:/g/personal/r_dhamrongsirivadh_alleninstitute_org/EexXIxeeIbNEs4KMjimcXOcBMn2J2QwxJhNEkOcRHC1eVg?e=JKa5WI)
-- From the link above, click the three dots on the top menu bar and select download. 
-- Open a file explorer and go to the Downloads folder. Use **Option 1** below. A prompt window should open and start installing. If this fails use **Option 2**. 
-  - **Option 1**: Double-click the file _install_napari.sh_
-  - **Option 2**: Search the file finder for Anaconda Prompt. Open version 3. Run the following commands one line at a time. 
+### 1. Install the plugin
+#### Windows
```
Comment (yrkim98) [inline README.md]: windows and mac should have the exact same install instructions

File: README.md [hunk 2]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
+- Open terminal
+- Create a virtual environment. We recommend using conda and python 3.10.
+  - To create a conda virtual environment, run the following commands one line at a time. 
     - conda create -n napari_annotator python=3.10 anaconda
```
Comment (yrkim98) [inline README.md]: terminal commands should be in code blocks

Comment (yrkim98) [inline README.md]: newest conda docs suggest we dont need the `anaconda` arugment when creating an env

File: README.md [hunk 3]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
-#### [Windows](https://alleninstitute-my.sharepoint.com/:u:/g/personal/r_dhamrongsirivadh_alleninstitute_org/EexXIxeeIbNEs4KMjimcXOcBMn2J2QwxJhNEkOcRHC1eVg?e=JKa5WI)
-- From the link above, click the three dots on the top menu bar and select download. 
-- Open a file explorer and go to the Downloads folder. Use **Option 1** below. A prompt window should open and start installing. If this fails use **Option 2**. 
-  - **Option 1**: Double-click the file _install_napari.sh_
-  - **Option 2**: Search the file finder for Anaconda Prompt. Open version 3. Run the following commands one line at a time. 
+### 1. Install the plugin
+#### Windows
+- Open terminal
+- Create a virtual environment. We recommend using conda and python 3.10.
+  - To create a conda virtual environment, run the following commands one line at a time.
```
Comment (yrkim98) [inline README.md]: nit: create and activate

File: README.md [hunk 4]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
     - conda create -n napari_annotator python=3.10 anaconda
     - conda activate napari_annotator
-    - python -m pip install --upgrade pip
+  - To create a venv virtual environment, run the following commands.
+    - python -m venv venv
+    - venv\Scripts\activate
+- Install Napari and the annotator plugin by running
     - python -m pip install "napari[all]"
     - python -m pip install napari-allencell-annotator
     - napari
```
Comment (yrkim98) [inline README.md]: this is to start napari via command line, shouldnt be under install

File: README.md [hunk 5]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
-- Run _chmod +x ./Downloads/install_napari.command_ 
-  - If you get a file not found error try adjusting the path to match where install_napari.command was downloaded.
-- Open finder, navigate to the file, double-click _install_napari.command_ . 
-  - A terminal window should open and start installing. 
-  
-
-### 3. Launch the Plugin
+- **Not working?** Try using conda forge instead of pip. 
+  - Ex: conda install -c conda-forge napari instead of python -m pip install "napari[all]"
+#### MacOS/Unix
```
Comment (yrkim98) [inline README.md]: would be good to add a step to say open the plugin and verify the annotator plugin was installed by going to the plugin tab

File: README.md [hunk 6]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
-### 2. Install the plugin
-Click the link corresponding to your OS.
-#### [Windows](https://alleninstitute-my.sharepoint.com/:u:/g/personal/r_dhamrongsirivadh_alleninstitute_org/EexXIxeeIbNEs4KMjimcXOcBMn2J2QwxJhNEkOcRHC1eVg?e=JKa5WI)
-- From the link above, click the three dots on the top menu bar and select download. 
-- Open a file explorer and go to the Downloads folder. Use **Option 1** below. A prompt window should open and start installing. If this fails use **Option 2**. 
-  - **Option 1**: Double-click the file _install_napari.sh_
-  - **Option 2**: Search the file finder for Anaconda Prompt. Open version 3. Run the following commands one line at a time. 
+### 1. Install the plugin
+#### Windows
+- Open terminal
```
Comment (yrkim98) [inline README.md]: not needed

File: README.md [hunk 7]
```diff
@@ -45,47 +45,44 @@ https://napari.org/plugins/index.html
...
+
+### 2. Launch the Plugin
 
 Once the napari window opens, go to **Plugins**.
 - If **napari-allencell-annotator** is listed click it to launch. 
 - If it is not listed 
 - **Install/Uninstall Plugins** ⇨ check the box next to **napari-allencell-annotator** ⇨ **close** ⇨ **Plugins** ⇨ **napari-allencell-annotator** .
 
-### 4. Re-opening the Plugin After Installing
+### 3. Re-opening the Plugin After Installing
```
Comment (yrkim98) [inline README.md]: this section is now irrelevant

**Review by yrkim98 — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (memeramita) [reply README.md]: removed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/3d3db184e562ea6100be66d48ae7ab1fe0a31e89

File: README.md (hunk 6 above)
Comment (memeramita) [reply README.md]: fixed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/3d3db184e562ea6100be66d48ae7ab1fe0a31e89

File: README.md (hunk 5 above)
Comment (memeramita) [reply README.md]: fixed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/a4037bd8bb1b4a5964fc6825d20050f2c1a03d93

File: README.md (hunk 4 above)
Comment (memeramita) [reply README.md]: I added a step to open Napari and verify the plugin is installed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/a4037bd8bb1b4a5964fc6825d20050f2c1a03d93

File: README.md (hunk 3 above)
Comment (memeramita) [reply README.md]: added in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/c574b97e4a818e5d310d60045241bc16295974f7

File: README.md (hunk 2 above)
Comment (memeramita) [reply README.md]: removed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/7bd29e078be68071a74b6e82cab6334730cb177e

Comment (memeramita) [reply README.md]: in code blocks now in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/3d3db184e562ea6100be66d48ae7ab1fe0a31e89

File: README.md (hunk 1 above)
Comment (memeramita) [reply README.md]: fixed in https://github.com/aics-int/napari-allencell-annotator/pull/108/commits/3d3db184e562ea6100be66d48ae7ab1fe0a31e89

**Pull request merged**